docs/area-owners.md

```diff
@@ -117,4 +117,4 @@ If you need to tag folks on an issue or PR, you will generally want to tag the o
 | area-TypeSystem-coreclr                        | @mangod9      | @davidwrighton @MichalStrehovsky @janvorli @mangod9 |                                                                                                                                                                                                                                                                                           |
 | area-UWP                                       | @tommcdon     | @nattress                                           | UWP-specific issues including Microsoft.NETCore.UniversalWindowsPlatform and Microsoft.Net.UWPCoreRuntimeSdk                                                                                                                                                                              |
 | area-VM-coreclr                                | @mangod9      | @mangod9                                            |                                                                                                                                                                                                                                                                                           |
-| area-VM-meta-mono                              | @marek-safar  | @lambdageek                                         |                                                                                                                                                                                                                                                                                           |
\ No newline at end of file
+| area-VM-meta-mono                              | @marek-safar  | @lambdageek                                         |                                                                                                                                                                                                                                                                                           |
```

Comment:
(nit) do you know what the diff is here?

Reply:
A newline was added to the end of the file.